drivers/sensor/ti_hdc/ti_hdc.h

```diff
@@ -9,23 +9,30 @@
 
 #include <kernel.h>
 
-#define TI_HDC_I2C_ADDRESS	0x40
+#define TI_HDC_REG_TEMP         0x0
+#define TI_HDC_REG_HUMIDITY     0x1
+#define TI_HDC_REG_CONFIG       0x2
 
-#define TI_HDC_REG_TEMP	0x0
-#define TI_HDC_REG_HUMIDITY	0x1
-#define TI_HDC_REG_MANUFID	0xFE
-#define TI_HDC_REG_DEVICEID	0xFF
+#define TI_HDC_MANUFID_REG      0xFE
```

Comment:
The schema was TI_HDC_REG_*, why you changed it for TI_HDC_REG_DEVICEID?
This also leads to unnecessary changes [here](https://github.com/zephyrproject-rtos/zephyr/pull/12616/commits/0f7e9a48482fecc6cd41b24cd7c375919e447550#diff-7e009466fba8f8e3e9e60aeb0c4f2856L121)